bdb/rep.c

```diff
@@ -2397,16 +2440,29 @@ static void got_new_seqnum_from_node(bdb_state_type *bdb_state,
             static unsigned long long count = 0;
             count++;
```

Comment:
We may want to atomic_inc(count);

Reply:
Doesn't have to be perfect- this is used only for wait_for_seqnum_trace flag